package.json

```diff
@@ -31,21 +31,23 @@
     "babel-plugin-template-html-minifier": "^4.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "eslint": "^7.9.0",
+    "eslint-config-prettier": "^7.2.0",
     "eslint-config-standard": "^16.0.2",
```

Comment:
[Standard](https://standardjs.com/) will probably conflict with Prettier in an annoying way, might want to swap it out for another ESLint configuration that is more focused on correctness than style.